python/lsst/analysis/tools/interfaces/datastore/_dispatcher.py

```diff
@@ -354,7 +354,8 @@ def _handleReferencePackage(self, meta: MutableMapping, bundle: MetricMeasuremen
                 except ValueError:
                     # Could not extract package timestamp leaving empty
                     pass
-        meta["reference_package"] = ref_package
+        # explicit handle if None was set in the bundle for the package
+        meta["reference_package"] = ref_package or ""
```

Comment:
Looks good, fields must be present in the payload even if the value is missing. An empty string is a good default value.